### PR TITLE
GMP Documentation: ActiveMQ

### PR DESCRIPTION
**Pull request by jsirianni**
This PR adds documentation configuration for the ActiveMQ GMP integration.

- Added documentation.yaml
- Added prometheus_metadata.yaml

### CODE REVIEW

File: integrations/activemq/prometheus_metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+platforms:
+  - type: GKE
+    launch_stage: GA
+    detections:
+      - characteristic_metric:
+          metric_type: prometheus.googleapis.com/activemq_connections_total/gauge
+    exporter_metadata:
+      name: ActiveMQ Prometheus Exporter
```
Comment (varun-c) [inline integrations/activemq/prometheus_metadata.yaml:8]: Based on the URL, "JMX Exporter" feels like a more appropriate name for this exporter, unless the doc url is for the wrong exporter. The bitnami exporter also does not seem to have a version 0.17.0. Did we mean to use this instead: https://github.com/prometheus/jmx_exporter?

**Review by varun-c — COMMENTED** [review]
prometheus_metadata.yaml looks good to me, except for the one comment. Adding YQ to review documentation.yaml.

File: integrations/activemq/prometheus_metadata.yaml (hunk 1 above)
Comment (jsirianni) [reply integrations/activemq/prometheus_metadata.yaml:8]: I think "JMX Exporter" makes sense. A handful of integrations rely on it. I will update.

Bitnami packages `prometheus/jmx_exporter` into a container image, and it does have tag `0.17.0`. The Bitnami readme links to the Prometheus repo. We can switch to the Prometheus link if you think that makes more sense.

File: integrations/activemq/documentation.yaml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  +     volumes:
+  +     - name: activemq-exporter
+  +       configMap:
+  +         name: activemq-exporter
+  +         items:
+  +         - key: config.yaml
+  +           path: config.yaml
+additional_install_info: |
+  These instructions assume you already have a working {{app_name_short}}
+  installation and want to modify it to include an exporter.
```
Comment (yqlu) [inline integrations/activemq/documentation.yaml:106]: Given that the template will already say "To install {{exporter_literal}} as a sidecar to {{app_name_short}}, modify your {{app_name_short}} configuration as shown in the following
example:"

Maybe this is redundant. 

Let's make the standard that we add a clarifying paragraph like this only when (as a bonus, as with the Redis case) followed immediately by additional detail for the other use case ("To set up {{app_name_short}} and {{exporter_name}} together from scratch, use this Helm chart...").

**Review by yqlu — APPROVED** [review]

File: integrations/activemq/prometheus_metadata.yaml (hunk 1 above)
Comment (varun-c) [reply integrations/activemq/prometheus_metadata.yaml:8]: We can keep the url as is if the bitnami jmx exporter is what we want users to be installing.

**Review by varun-c — APPROVED** [review]

**Review by yqlu — APPROVED** [review]

**Pull request merged**